src/utils/index.ts

```diff
@@ -134,9 +134,9 @@ export function readPackageJson (
 
 export function resolveAliases (_aliases: Record<string, string>) {
   // Sort aliases from specific to general (ie. fs/promises before fs)
-  const aliases = Object.fromEntries(Object.entries(_aliases).sort(([a], [b]) => {
-    return b.split('/').length - a.split('/').length
```

Comment:
`(b.split('/').length - a.split('/').length) || (b.length - a.length)`

This should have same effect :D

Reply:
`['a/foo', 'a/foo-bar'].sort((a, b) => b.split('/').length - a.split('/').length || b.length - a.length)`